README.md

```diff
@@ -39,3 +39,13 @@ pulpcore z-release, the existing y-release branch is built and published again.
 
 * Go to the y-release branch you're releasing for and make sure the pulp/Containerfile looks good.
 * Kick off a new build at [the publish workflow](https://github.com/pulp/pulp-oci-images/actions/workflows/publish_images.yaml)
+
+# Debugging
+
+## Debugging the initializtion scripts
+
+To debug the scripts under `/etc/cont-init.d/`, rather than running the usual command to start the container, run a command
+like the following (no "--detach", with "-ti", with "/bin/sh" on the end.)
+```bash
+docker run -ti --name pulp --publish 8080:443 --volume "/$(pwd)/settings:/etc/pulp:Z" --device /dev/fuse pulp/pulp /bin/sh
+```
```

Comment:
Do you then need to run the scripts by hand? Maybe an example would be beneficial.

Reply:
I updated the instructions.